apps/public-docsite-v9/src/Concepts/Migration/FromV0/Components/Flex.stories.mdx

```diff
@@ -0,0 +1,113 @@
+import { Meta } from '@storybook/addon-docs';
```

Comment:
This is really confusing, we have multiple Flex migration pages

![image](https://user-images.githubusercontent.com/20744592/221863327-b657a075-13d7-44e8-a0a7-b81b312397f9.png)